src/components/Dashboard/Dashboard.jsx

```diff
@@ -5,7 +5,7 @@ import { Avatar } from "@material-ui/core";
 import "./styles.css";
 import Post from "./Post/Post";
 
-import React, {useContext} from "react";
+import React, { useContext, useState } from "react";
```

Comment:
For convenience, any imports from inside the project come last and those from dependencies come first! This should be the first line!

Reply:
Done 140ca9f9a8e00f3dd1f9e0803e6fff7bbca30740